test/cpp_extensions/libtorch_agnostic_extension/libtorch_agnostic/ops.py

```diff
@@ -487,3 +487,17 @@ def test_get_num_threads() -> int:
     Returns: int - the number of threads for the parallel backend
     """
     return torch.ops.libtorch_agnostic.test_get_num_threads.default()
+
+
+def mv_tensor_accessor(t1, t2) -> Tensor:
+    """
+    Returns matrix-vector product.
+
+    Args:
+        t1: any 2-D Tensor
+        t2: any 1-D Tensor
```

Comment:
there are shape constraints, no? That if (M, N) is the size of t1, t2 must be at least N long?